docs/participate/treasury-proposal.md

```diff
@@ -0,0 +1,53 @@
+---
+id: treasury-proposal
+title: Create a treasury proposal
+---
+
+Complete these steps to create a well-formed Treasury proposal
+
+# Discuss
+
+The first step in applying for a Treasury grant is to join the community in the [Element Direction Channel](https://matrix.to/#/#kilt-direction:matrix.org), [Discord Treasury Channel](https://discord.gg/nUpqDfQ6kJ) or on [Telegram](https://t.me/KILTProtocolChat) to discuss and share your intended proposal. This will help you get valuable community feedback throughout the process. This also gives Council members an open and transparent way to measure community sentiment.  
```

Comment:
1. Unfortunately comms have not updated the site as requested (and approved) 2022-05-10. Instead of/additional to channeling folks to Element, Discord, or Telegram, we wanted to add [Polkassembly Discussion](https://kilt.polkassembly.network/discussions) to the list. We devs and Council would like to have all "formal" discussions on Polkassembly. Discord should only serve as an option to "brainstorm". However, once they have a proposal draft, we want it to be posted to Polkassembly
2. New line please
```suggestion
The first step in applying for a Treasury grant is either to join the community in the [Discord Treasury Channel](https://discord.gg/nUpqDfQ6kJ) to brainstorm the scope of your proposal intention or immediately open a discussion on [Polkassembly](https://kilt.polkassembly.network/discussions). 
This will help you get valuable community feedback throughout the process.
It also gives Council members an open and transparent way to measure community sentiment.  
```